user-management-spec.json

```diff
@@ -110,6 +110,53 @@
                 }
             }
         },
+        "/api/v1/refresh-token": {
+            "post": {
+                "tags": [
+                    "User Management"
+                ],
+                "summary": "Refresh the user token",
+                "description": "Refresh the user token",
+                "operationId": "refresh-token",
+                "parameters": [
+                    {
+                        "name": "refresh_token",
+                        "in": "header",
+                        "description": "Valid refresh token",
+                        "required": true,
+                        "schema": {
+                            "type": "string"
+                        }
+                    }
+                ],
+                "responses": {
+                    "400": {
+                        "description": "Invalid request."
+                    },
+                    "200": {
+                        "description": "Successful response - Returns the access token object.",
+                        "content": {
+                            "application/json": {
+                                "schema": {
+                                    "type": "array",
+                                    "items": {
+                                        "$ref": "#/components/schemas/Token"
+                                    }
+                                }
+                            }
+                        }
+                    },
+                    "500": {
+                        "description": "Internal server error."
+                    }
+                },
+                "security": [
+                    {
+                        "AuthorizationToken": []
```

Comment:
Does refresh token also need security authorization?

Reply:
not required it does not check in code. I will update the spec